dist/vl-header.js

```diff
@@ -76,8 +75,45 @@ export class VlHeader extends vlElement(HTMLElement) {
     if (!VlHeader.header) {
       document.body.insertAdjacentHTML('afterbegin', this.getHeaderTemplate());
     }
+    this.__observeHeaderElementIsAdded();
     eval(code.replace(/document\.write\((.*?)\);/, 'document.getElementById("' + VlHeader.id + '").innerHTML = $1;'));
-    this.dispatchEvent(new CustomEvent(VlHeader.EVENTS.ready));
+  }
+
+  __observeHeaderElementIsAdded() {
+    const target = document.querySelector('#' + VlHeader.id);
+    const headerObserver = new MutationObserver((mutations, observer) => this.__headerObserverCallback(mutations, observer));
+    const config = {attributes: false, childList: true, characterData: false};
+    headerObserver.observe(target, config);
+  }
+
+  __headerObserverCallback(mutations, observer) {
+    mutations.forEach((mutation) => {
```

Comment:
gaat ge nu niet per mutatie dit event krijgen?
moet ge ook niet bijhouden of ge het event al eens gestuurd hebt?

Reply:
normaal gezien gaat de observer stoppen eenmaal het element gevonden is (de observer.disconnect enkele lijnen lager). Net erboven wordt het event verstuurt. Aangezien de observer stopt, zal er ook maar één keer een event verstuurd worden.

Reply:
dat hangt er dan deels van af. als ge in 1x meer dan 1 mutatie doorkrijgt, doet ge bij de eerste disconnect maar uwe forEach loop gaat nog wel verder.
dus misschien interessanter dat ge doet.
`
if (__vlHeaderIsToegevoegdInEenVanDeMutaties___) {
  stuurEvent()
  disconnect();
}
`

zo zit de loop over de mutaties in de if-check, en gaat ge daarna zeker geen mutaties meer door krijgen?